database_sqlite.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"database/sql"
+	"log"
+	"os"
+	"time"
+)
+
+type SqliteDatabase struct {
+	*sql.DB
+}
+
+func sqliteInit(dbParams map[string]interface{}) (*SqliteDatabase, error) {
+
+	dbFilename, ok := dbParams["file"].(string)
+	if !ok {
```

Comment:
errrr, don't we want to raise an error instead of silently creating a new database? This could be a problem:

 - the user might already have a commento db and we'd be loading a new one; all the comments on their website will disappear
- they might have something else in `sqlite3.db` and we could possibly overwrite it!

Always raise errors when in doubt :)